### DISCUSSION TITLE
Search by address

### DISCUSSION THREAD

**Discussion by JavierPorron** [Congyuwang/Rusty-Bitcoin-Explorer#23] [Q&A]
Is it posibble to do a search getting tx from an address and searching by tx of that address? 
Thanks for reading.

**Comment by Congyuwang** [comment]
It is, but this is not a feature of this crate. Nor does the bitcoin Core data structure offer a convenient way to do that. So, you probably need to loop over all blocks and build a new database with addresses as index or something similar.

**Comment by JavierPorron** [comment]
Thanks for answer. And follow Tx or something similar?  I see the "Fconnectedtransaction" struct that I don't know if could help me on what I want or I know that it's possible iterate the tx inside a block but it's not what I am looking....
My goal finally is follow an amount of bitcoins making a graph with the tx.
Sorry for the inconveniences

**Comment by agemuend** [comment]
I think you might want to read these articles: 

https://en.bitcoin.it/wiki/Invoice_address
https://en.bitcoin.it/wiki/From_address
https://en.bitcoin.it/wiki/Address_reuse
